test/e2e/plugin.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
```

Comment:
@Kavinjsir we should not add a new file here. 
In this case, we need only in the Kubebuilder Makefile to call the test.

We would only need to execute this test once. However, to make simple let's call it with the others and a TODO here to sort it out after. 

That means:
a) Remove this file plugin.sh
b) Add the new test in : https://github.com/kubernetes-sigs/kubebuilder/blob/master/test/e2e/setup.sh#L65
c) Revert the changes made in the Kubebuilder Makefile

Reply:
c/c @varshaprasad96 

/hold

Reply:
Sounds good to me. I've updated the code based on the steps and it looks work well.